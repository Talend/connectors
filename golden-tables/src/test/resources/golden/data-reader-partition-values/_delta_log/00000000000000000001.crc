{"tableSizeBytes":3623,"numFiles":3,"numMetadata":1,"numProtocol":1,"numTransactions":0}
