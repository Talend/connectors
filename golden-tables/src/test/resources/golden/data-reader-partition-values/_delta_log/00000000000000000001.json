{"commitInfo":{"timestamp":1630660230918,"operation":"WRITE","operationParameters":{"mode":"Append","partitionBy":"[]"},"readVersion":0,"isolationLevel":"WriteSerializable","isBlindAppend":true,"operationMetrics":{"numFiles":"3","numOutputBytes":"3623","numOutputRows":"3"}}}
{"add":{"path":"outofdate=false/part-00000-d2351dfd-6272-4004-b2ec-354e515c5bcd.c000.snappy.parquet","partitionValues":{"outofdate":"false"},"size":1203,"modificationTime":1630660230000,"dataChange":true,"stats":"{\"numRecords\":1,\"minValues\":{\"id\":\"1\",\"name\":\"Fords\",\"release\":\"2012-08-20\",\"sales\":\"5000000\"},\"maxValues\":{\"id\":\"1\",\"name\":\"Fords\",\"release\":\"2012-08-20\",\"sales\":\"5000000\"},\"nullCount\":{\"id\":0,\"name\":0,\"release\":0,\"sales\":0}}"}}
{"add":{"path":"outofdate=false/part-00001-5ced1367-28f8-4dfb-abf7-6827b3ba657d.c000.snappy.parquet","partitionValues":{"outofdate":"false"},"size":1211,"modificationTime":1630660230000,"dataChange":true,"stats":"{\"numRecords\":1,\"minValues\":{\"id\":\"2\",\"name\":\"Tesla\",\"release\":\"2019-07-20\",\"sales\":\"40000000\"},\"maxValues\":{\"id\":\"2\",\"name\":\"Tesla\",\"release\":\"2019-07-20\",\"sales\":\"40000000\"},\"nullCount\":{\"id\":0,\"name\":0,\"release\":0,\"sales\":0}}"}}
{"add":{"path":"outofdate=true/part-00002-b2356873-af54-495b-823f-bf9a784a2be8.c000.snappy.parquet","partitionValues":{"outofdate":"true"},"size":1209,"modificationTime":1630660230000,"dataChange":true,"stats":"{\"numRecords\":1,\"minValues\":{\"id\":\"3\",\"name\":\"Mercedes-Benz\",\"release\":\"2006-06-10\",\"sales\":\"\"},\"maxValues\":{\"id\":\"3\",\"name\":\"Mercedes-Benz\",\"release\":\"2006-06-10\",\"sales\":\"\"},\"nullCount\":{\"id\":0,\"name\":0,\"release\":0,\"sales\":0}}"}}
